keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -590,10 +595,12 @@ const (
 type ServiceImportSpec struct {
   // +listType=atomic
   Ports []ServicePort `json:"ports"`
-  // +kubebuilder:validation:MaxItems:=1
+  // +kubebuilder:validation:MaxItems:=2
   // +optional
   IPs []string `json:"ips,omitempty"`
   // +optional
+  IPFamilies []corev1.IPFamily `json:"ipFamilies,omitempty"`
+  // +optional
```

Comment:
I was not following this conversation so apologies if this was already discussed, but this was very challenging to implement in Kube, cc @thockin as he did all the work, because `IPs` and `IPFamilies` are related, and the defaulting and validation or updates got very tricky

Reply:
if the goal is to mirror the services fields why don't you add a reference? is to avoid the indirection layer?